docs: tweaks for translations

naomi-lgbt opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #40947

<!-- Feel free to add any additional description of changes below this line -->

- Tweaks some verbiage to facilitate translations
- Updates chat URLs to include `channels` path.

Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

/cc @RandellDawson

Commit:
docs: tweaks for translations

- Tweaks some verbiage to facilitate translations
- Updates chat URLs to include `channels` path.

Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

ojeytonwilliams reviewed (changes requested):
It's a shame we have to warp the text to make it translatable.  Would it be reasonable to mark certain code blocks, like 
```
`https://github.com/YOUR_USER_NAME/freeCodeCamp`
```
as being translatable?  That should allow us to write the best version we can of the original and give the translators sufficient freedom to recreate it.

For this particular case we could go with 

```
`https://github.com/<YOUR_USER_NAME>/freeCodeCamp`
```

to emphasise that `<YOUR_USER_NAME>` is just a placeholder to be replaced.  I think that's a fairly standard convention.

ojeytonwilliams commented:
> Yes, we could avoid hiding that particular string, but allowing translations would cause our QA check to throw an error about translating in-line code blocks.

My mistake, I was thinking about triple backtick'd code blocks - we don't hide inline code at all, just QA as you say.

I guess we could write more convoluted QA to detect code blocks we do want translating, but that feels like a mistake.

It also feels like the Glossaries should help i.e. add `YOUR_USER_NAME` to the glossary so that we can tell translators that it's fine to ignore the QA here, but it doesn't seem to pick it up.

Commit:
docs: apply suggestion

Removes `English` from string regarding how to run tests on specific challenge.

Co-authored-by: Oliver Eyton-Williams <ojeytonwilliams@gmail.com>

nhcarrigan commented:
We are also running in to an issue with Crowdin not parsing the quadruple-backticks correctly (so the challenge file template is a mess when viewed on Crowdin)

ojeytonwilliams reviewed (dismissed):
LGTM :+1: 

Is there more you wanted to do, Nick?  I know this is a draft, but I'd be happy to merge as is.

nhcarrigan commented:
This could be merged as is - we still need to figure out the quadruple backtick issue, though.

nhcarrigan commented:
@ojeytonwilliams This PR also included fixes for our chat links, so I'm going ahead with marking it ready for review.

We can address the quadruple-backtick issue in a separate PR, perhaps after soliciting a recommendation from Crowdin.

nhcarrigan commented on docs/how-to-help-with-video-challenges.md:
```diff
@@ -85,8 +85,8 @@ videoId: nVAaxZ34khk
 
-Next, access the YouTube video with that videoId. The URL for the video will be:
-https://www.youtube.com/watch?v=[videoId]    (add videoId to the URL without square brackets)
+Next, access the YouTube video with that videoId. The url for the video will be:
```
```suggestion
Next, access the YouTube video with that videoId. The URL for the video will be:
```

naomi-lgbt reviewed (commented):
quick comment for change missed in rebase

Commit:
fix(docs): missed correction in rebase

RandellDawson commented on docs/how-to-help-with-video-challenges.md:
````diff
@@ -122,7 +122,7 @@ More answers
 ```
 
-Update the  “Question Text” with your question. Update the `Answer 1`, `Answer 2`, and so on with the possible answers. Make sure to update the video-solution number with the correct answer number. You can add more possible answers using the same format. The question and answers can be surrounded by quotation marks.
+Update the  “Question Text” with your question. Update the `Answer 1`, `Answer 2`, and so on with the possible answers. Make sure to update the `video-solution` number with the correct answer number. You can add more possible answers using the same format. The question and answers can be surrounded with quotation marks.
````
@nhcarrigan What do you think about the following or something like it?  I was trying to avoid "Question Text".
```suggestion
Add/Update the  question under `# --question--` `## --text--`. Add/Update answers (`Answer 1`, `Answer 2`, and so on) under `## --answers--`. Make sure to update the number under `## --video-solution--` with the correct answer number. You can add more possible answers using the same format. The question and answers can be surrounded with quotation marks.
```

RandellDawson commented on docs/how-to-help-with-video-challenges.md:
````diff
@@ -86,7 +86,7 @@ videoId: nVAaxZ34khk
 ```
 
 Next, access the YouTube video with that videoId. The URL for the video will be:
````
```suggestion
Next, access the YouTube video with that `videoId`. The URL for the video will be:
```

Also, above this, the following appears:
> For example, in the following code from the header of a video challenge markdown file, the videoId is "nVAaxZ34khk". On GitHub, the information should be laid out in a table format.

It would be nice to add backticks around `videoId`.

RandellDawson commented on docs/how-to-setup-freecodecamp-locally.md:
```diff
@@ -68,15 +68,15 @@ This is essential, as it allows you to work on your own copy of freeCodeCamp on
 > The main repository at `https://github.com/freeCodeCamp/freeCodeCamp` is often referred to as the `upstream` repository.
 >
-> Your fork at `https://github.com/YOUR_USER_NAME/freeCodeCamp` is often referred to as the `origin` repository.
+> Your fork at `https://github.com/YOUR_USER_NAME/freeCodeCamp` is often referred to as the `origin` repository. `YOUR_USER_NAME` would be replaced with your GitHub user name.
```
```suggestion
> Your fork at `https://github.com/YOUR_USER_NAME/freeCodeCamp` is often referred to as the `origin` repository. `YOUR_USER_NAME` would be replaced with your GitHub username.
```

RandellDawson commented on docs/how-to-setup-freecodecamp-locally.md:
````diff
@@ -138,7 +138,7 @@ You need a reference from your local clone to the `upstream` repository in addit
    ```
 
-   The output should look something like below:
+   The output should look something like below (replacing `YOUR_USER_NAME` with your GitHub Username):
````
```suggestion
   The output should look something like below (replacing `YOUR_USER_NAME` with your GitHub username):
```

RandellDawson commented on docs/how-to-work-on-coding-challenges.md:
````diff
@@ -472,10 +471,10 @@ You are also able to test one challenge individually by performing the following
    ```
 
-2. Run the following for each challenge file for which you have changed:
+2. Run the following for each challenge file for which you have changed (replacing `title` with the full title of the challenge):
````
```suggestion
2. Run the following for each challenge file for which you have changed (replacing `challenge-title-goes-here` with the full title of the challenge):
```

RandellDawson commented on docs/how-to-work-on-coding-challenges.md:
````diff
@@ -472,10 +471,10 @@ You are also able to test one challenge individually by performing the following
 
    ```
-   npm run test -- -g 'the full English title of the challenge'
+   npm run test -- -g 'title'
````
```suggestion
   npm run test -- -g challenge-title-goes-here
```

RandellDawson reviewed (changes requested):
@nhcarrigan Changes look good for the most part.  I have a few suggestions.  You may or may not choose to use them.

Commit:
fix(docs): apply review suggestions

Co-authored-by: Randell Dawson <5313213+RandellDawson@users.noreply.github.com>

RandellDawson commented:
@nhcarrigan Can you resolve the merge conflict?

Commit:
Merge branch 'main' into docs/refactor
